README.md

```diff
@@ -11,32 +11,34 @@ _Having problems? want to contribute? join our [community slack](http://devtools
 Istanbul's state of the art command line interface, with support for:
 
 * applications that spawn subprocesses.
-* ES2015 transforms, via [`babel-plugin-istanbul`], or source-maps.
+* source mapped coverage of Babel and TypeScript projects
 
 ## Installation & Usage
 
-You can use `npx` instead of installing nyc as a dependency.
-Simply add `nyc@latest` before your test runner is invoked (replace `mocha` with your test runner everywhere you see it):
-
+Use your package manager to add it as a dev dependency: `npm i -D nyc` or `yarn add -D nyc`.
+You can use nyc to call npm scripts (assuming they don't already have nyc executed in them), like so (replace `mocha` with your test runner everywhere you see it):
 ```json
 {
   "scripts": {
-    "test": "npx nyc@latest mocha"
+    "test": "mocha",
+    "coverage": "nyc npm run test"
   }
 }
 ```
 
-Or use your package manager to add it as a dev dependency: `npm i -D nyc` or `yarn add -D nyc`.
-You can also use nyc to call npm scripts (assuming they don't already have nyc executed in them), like so:
+You can use also `npx` instead of installing nyc as a dependency, but you might get updates you are not ready for; to get around this, pin to a specific major version by specifying, e.g. `nyc@14`.
+In general, simply add `nyc` before your test runner is invoked:
```

Comment:
simply add `nyc`..?  Should this say `npx nyc@latest`?  Maybe just remove this line?

Other than this line it everything looks good.